easybuild/easyconfigs/t/tbb/tbb-2019_U9-GCCcore-8.3.0.eb

```diff
@@ -1,18 +1,15 @@
 name = 'tbb'
 version = '2019_U9'
 
-homepage = 'https://github.com/oneapi-src/oneTBB'
+homepage = 'https://01.org/tbb/'
 description = """Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
  take full advantage of multicore performance, that are portable, composable and have future-proof scalability."""
 
 toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
 
-source_urls = ['https://github.com/oneapi-src/oneTBB/archive/']
+source_urls = ['https://github.com/01org/tbb/archive/']
 sources = ['%(version)s.tar.gz']
-checksums = [
-    ('15652f5328cf00c576f065e5cd3eaf3317422fe82afb67a9bcec0dc065bd2abe',
-     '3f5ea81b9caa195f1967a599036b473b2e7c347117330cda99b79cfcf5b77c84'),
-]
+checksums = ['3f5ea81b9caa195f1967a599036b473b2e7c347117330cda99b79cfcf5b77c84']
```

Comment:
This checksum was already included? You just removed the old one (which is also correct for the older, identical sources)